.github/workflows/dockerimage.yml

```diff
@@ -1,22 +1,22 @@
-
 name: Run tests
 
 on:
-
   push:
     branches: [ master ]
 
   pull_request:
     branches: [ master ]
 
-
 jobs:
 
   test-on-ubuntu:
-
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
-    - uses: technote-space/get-diff-action@v1
-    - name: Get the list of modified files only
-      run: pip install toml pytablewriter && python generate.py ${{ env.GIT_DIFF }} && docker build -t shellspec . && docker run -v $PWD:/app shellspec bash -c "cd /app && ./test.sh ${{ env.GIT_DIFF }}"
+      - uses: actions/checkout@v2
+      - uses: technote-space/get-diff-action@v1
+      - name: Run tests for the modified/ added files
+        run: |
+          pip install toml pytablewriter
+          python generate.py ${{ env.GIT_DIFF }}
+          docker build -t shellspec .
```

Comment:
Let's move this Docker image to GitHub Packages and put up a GitHub Action to build it and push. We can simply pull the image here.

Reply:
Let's take this into this week https://github.com/leopardslab/installer.to/issues/59